snap/snapcraft.yaml

```diff
@@ -261,6 +263,7 @@ parts:
       - postgresql-16-pgvector
       - postgresql-16-pgaudit
       - postgresql-16-set-user
+      - postgresql-16-login-hook
```

Comment:
Package seems to be called `postgresql-login-hook`.

Reply:
from launchpad:

```
Built 
postgresql-16-login-hook Postgres database extension to execute some code on user 
postgresql-16-login-hook-dbgsym debug symbols for postgresql-16-login-hook
```

looks like the CI is failing because the package failed to build in launchpad for arm64. triggered a rebuild :crossed_fingers: